7.7.Поиск_ошибок_и_ревью_кода/7.7.5.py

```diff
@@ -0,0 +1,6 @@
+s = 0
+for i in range(7):
+    n = int(input())
+    if n % 2 == 0:
+        s +=n
+print(s)
```

Comment:
Все хорошо, но лучше отделять логические части пустыми строками:
```suggestion
s = 0

for i in range(7):
    n = int(input())
    if n % 2 == 0:
        s +=n

print(s)
```